src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -827,14 +827,12 @@ else if (!referencePath.isDirectory())
                             listener.getLogger().println("[WARNING] Reference path does not contain an objects directory (not a git repo?): " + objectsPath);
                         else {
                             File alternates = new File(workspace, ".git/objects/info/alternates");
-                            try (PrintWriter w = new PrintWriter(alternates, Charset.defaultCharset().toString())) {
+                            try (PrintWriter w = new PrintWriter(alternates, Charset.defaultCharset())) {
```

Comment:
Using newer Java Platform functionality that avoids the messiness around `UnsupportedEncodingException`.